pkg/orchestrator/pod.go

```diff
@@ -112,9 +112,9 @@ func ProcessPodlist(podList []*v1.Pod, groupID int32, cfg *config.AgentConfig, h
 }
 
 // ScrubContainer scrubs sensitive information in the command line & env vars
-func ScrubContainer(c *v1.Container, cfg *config.AgentConfig) {
+func ScrubContainerOld(c *v1.Container, cfg *config.AgentConfig) bool {
```

Comment:
as long as you add a release note you can remove the old function, if we need it we'll bring it back from git.

Reply:
Sure! This is mainly here to have the function for benchmarking purposes.